Readme Update with Logo

Chikdolman opened the pull request:
Add award-winning Castanets logo image
Two versions of logos are added.
One alpaca image with black circle
The other alpaca image without the black circle

Readme.MD also modified to make it more developer-friendly

Chikdolman committed:
Castanets Logo Images (SVG)

Add award-winning Castanets logo image
Two versions of logos are added.
One alpaca image with black circle
The other alpaca image without the black circle

Chikdolman committed:
Update Readme.md

modified Readme.md
1. Added Logo
2. Added Android Build
3. Added Castanets explanation

Chikdolman committed:
Update Android execution method

Update Android execution method

Sunwoo-Nam commented on README.md:
````diff
@@ -11,135 +23,159 @@ Clone the depot_tools repository:
-Device B: Utility Process (Network Service)
+_Device B: Renderer Process_
+```sh
+$ out/Default/chrome --type=renderer --enable-castanets=<BROWSER IP ADDR>
````
It doesn't support --enable-castanets in chromium_69.
--server-address=<BROWSER IP ADDR"

Chikdolman replied:
Thank you. I missed that. Now modified.

djmixkim commented on README.md:
````diff
@@ -11,135 +23,159 @@ Clone the depot_tools repository:
+Build castanets (the “chrome” target) with Ninja using the command:
 ```sh
-$ gn gen out/Default
+$ ninja -C out/Default chrome
````
How about use autoninja instead of ninja?
It will provides optimal values automatically.

- https://chromium.googlesource.com/chromium/src/+/master/docs/android_build_instructions.md
- https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/refs/heads/master/autoninja.py

Chikdolman replied:
Changed to autoninja. Thank you

Chikdolman committed:
Update README.md

ninja build --> autoninja build for optimization
--enable-castanets --> --server-address for option support

Sunwoo-Nam reviewed (approved):
Looks good to me.
